### PR TITLE
[RA2]: Add separate build capability

### PR DESCRIPTION
**Pull request by CsatariGergely**
This change makes it possible to build only RA2 with a `tox -e docs` command in the directory of the sub-broject.

### CODE REVIEW

File: doc/ref_arch/kubernetes/conf.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
+import sphinx_material
+
+project = 'Anuket Reference Implementation for Kubernetes'
```
Comment (pgoyal01) [inline doc/ref_arch/kubernetes/conf.py:3]:
```suggestion
project = 'Anuket Reference Architecture for Kubernetes'
```

File: doc/ref_arch/kubernetes/conf.py [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+html_title = "Anuket Reference Implementation for Kubernetes"
```
Comment (pgoyal01) [inline doc/ref_arch/kubernetes/conf.py:4]:
```suggestion
html_title = "Anuket Reference Architecture for Kubernetes"
```

**Comment by pgoyal01** [comment]
@CsatariGergely You have asked for a review but this is still in "Draft"

File: doc/ref_arch/kubernetes/conf.py (hunk 1 above)
Comment (rgstori) [reply doc/ref_arch/kubernetes/conf.py:3]: +1

File: doc/ref_arch/kubernetes/conf.py (hunk 2 above)
Comment (rgstori) [reply doc/ref_arch/kubernetes/conf.py:4]: +1

**Comment by CsatariGergely** [comment]
> @CsatariGergely You have asked for a review but this is still in "Draft"

Yes and thanks for the suggestions. I will undraft this once we have a solution for linking what works for both the central and the local builds.

**Comment by collivier** [comment]
They are lots of md files listed in RA2 which should be removed - grep ".md" chapters/*.md

Once may have been removed since you started working on this PR
CNTT/doc/ref_arch/kubernetes/chapters/chapter01.rst:61:broken link: ../../../common/policies.md#cntt-policies-for-managing-non-conforming-technologies ()

Else I'm fine with the target here.

**Review by gkunz — APPROVED** [review]

**Comment by collivier** [comment]
obsolete?

**Comment by rgstori** [comment]
closed as superseded by other PR #2882

**Pull request closed without merging**